.changeset/chubby-cameras-argue.md

```diff
@@ -0,0 +1,11 @@
+---
+"@biomejs/biome": major
+---
+
+Removed `--config-path` argument from `biome lsp-proxy` and `biome start` commands.
+
+If you are using one of our official plugins for IDEs or editors, just update it to the latest version.
```

Comment:
That's a tricky one, but I believe this upgrade isn't going to be backwards compatible, so we should make sure we spell it out loud to our users.

Meaning that using the latest of a plugin will require V2 🤔. Is that correct?

Reply:
@ematipico It depends on how the plugin handles the upgrade. If the plugin see `package.json` or `biome --version`, it can know Biome is >=2 or not and change the behaviour.